applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_linear_nodal_extrapolator.cpp

```diff
@@ -12,16 +12,41 @@
 //
 
 #include "custom_utilities/linear_nodal_extrapolator.h"
+#include "custom_utilities/ublas_utilities.h"
 #include "geometries/quadrilateral_2d_4.h"
 #include "geometries/quadrilateral_2d_8.h"
 #include "geometries/triangle_2d_3.h"
 #include "geometries/triangle_2d_6.h"
 #include "tests/cpp_tests/geo_mechanics_fast_suite.h"
 #include <boost/numeric/ublas/assignment.hpp>
+#include <numbers>
 
 namespace Kratos::Testing
 {
 
+KRATOS_TEST_CASE_IN_SUITE(NodalExtrapolator_GivesCorrectExtrapolationMatrix_For2D3NLine,
+                          KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    Line2D3<Node> geometry(Kratos::make_intrusive<Node>(1, 0.0, 0.0, 0.0),
+                           Kratos::make_intrusive<Node>(2, 1.0, 0.0, 0.0),
+                           Kratos::make_intrusive<Node>(3, 0.5, 0.0, 0.0));
+
+    const LinearNodalExtrapolator nodal_extrapolator;
+
+    constexpr auto integration_method = GeometryData::IntegrationMethod::GI_GAUSS_2;
+    auto           extrapolation_matrix =
+        nodal_extrapolator.CalculateElementExtrapolationMatrix(geometry, integration_method);
+
+    // clang-format off
+    Matrix expected_extrapolation_matrix = UblasUtilities::CreateMatrix(
```

Comment:
Let's make this one `const auto`:
```suggestion
    const auto expected_extrapolation_matrix = UblasUtilities::CreateMatrix(
```

Reply:
Done